content/docs/12-enterprise-version/Helm Chart Install References

```diff
@@ -0,0 +1,161 @@
+---
+title: "Helm Chart Install References"
+metaTitle: "Helm Chart Install References"
+metaDescription: "Reference page for Pallete Helm chart self-installation values.yaml file parameters."
+icon: ""
+hideToC: true
+fullWidth: true
+---
+
+# Helm Chart Install Reference Guide
+You can install the Palette Helm Chart in a multi-node Kubernetes cluster for use in production environments. The [Self-hosted Palette installation using Helm](https://docs.spectrocloud.com/enterprise-version/deploying-palette-with-helm) allows users to customize as per their needs by specifying user-defined values in the **values.yaml file.**
+This reference guide explains the parameters available in the **values.yaml** file from the Helm Chart for your installation.
+
+## Mandatory parameters
+
+The following parameters are mandatory for the self-hosted Palette installation using Helm. 
+
+- **env.rootDomain**- Used to configure the domain for the Palette installation. **Note:** You should create a DNS record separately, and it should be a wildcard to account for Organization prefixes. Review the [environment parameters](#environment-parameters) to learn more. 
+
+- **natsUrl**- The URL format specifies configuring the NATS server(s) to the IP address and port. Review the [Network Address Translation (NATS) parameters](#network-address-translationnats-parameters) to learn more. 
+ 
+- Refer to the [Registry and Scar parameters](#registry-and-scar-parameters) used to specify the Docker registry where the chart images are stored and the Spectro Cloud Admission Registrar (SCAR), respectively.
+
+## MongoDB parameters
+
+The below table outlines the values.yaml parameters for configuring a **MongoDB** deployment. 
+
+| **Parameters** | **Default value** | **Type** | **Description** | **Required/Optional** |
+| --- | --- | --- | --- | --- |
+| `internal` | N/A | Boolean | Specifies the **MongoDB** deployment either in-cluster or using **Mongo Atlas**. | Required |
+| `databaseUrl` | `mongo-0.mongo,mongo-1.mongo,mongo-2.mongo` | String | **MongoDB** URL. Change if using **Mongo Atlas**.| Required|
+| `databasePassword` | "" | String | **Mongo Atlas** database password, base64-encoded. | Optional |
```

Comment:
The description should keep it generic. Such as "The base64 encoded MongoDB password".